pom.xml

```diff
@@ -26,6 +26,11 @@
             <version>1.2.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.141.59</version>
```

Comment:
let's use latest stable version [4.10.0](https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java/4.10.0)
stable means version that passed all release phases and was tested. Now it is 4.10.0

Reply:
fixed